frontend/src/components/recommendation/map/Map.tsx

```diff
@@ -60,10 +75,11 @@ const Map = () => {
           }}
         >
           {allLocations.map((location, index) => (
-            <MarkerF
+            <Marker
```

Comment:
Can I ask why you want to use Marker instead of MarkerF?
I found the bug before, that [when I use `<Marker/>`, it doesn't appear in lcoalhost.](https://stackoverflow.com/questions/72112491/marker-not-showing-react-google-maps-api-on-localhost-next-js)